components/yaml/introduction.rst

```diff
@@ -148,6 +148,21 @@ error if something goes wrong by adding the filename to the message.
 
 .. _components-yaml-dump:
 
+Objects for Mappings
+....................
+
+.. versionadded:: 2.7
```

Comment:
So it was added to `Parser#parse` in `2.6` but to `Yaml::parse` in 2.7. Kept the note at 2.7 as I guess `Yaml` is the main entry point.